src/agent.cpp

```diff
@@ -948,8 +975,13 @@ bool Agent::doMarkPlayer(int unum) {
   }
   double x = player_pos.x + (kicker_pos.x - player_pos.x)*0.1;
   double y = player_pos.y + (kicker_pos.y - player_pos.y)*0.1;
-  Body_GoToPoint(Vector2D(x,y), 0.25, ServerParam::i().maxDashPower()).execute(this);
-  return true;
+  bool may_fix = wm.self().collidesWithPost();
+
+  if (Body_GoToPoint(Vector2D(x,y), 0.25, ServerParam::i().maxDashPower()).execute(this)) {
```

Comment:
if (Body_GoToPoint(...).execute(this) || wm.self().collidesWithPost()) {
return true;
}
return false;